conda/recipes/jupyterlab-nvdashboard/meta.yaml

```diff
@@ -17,18 +17,19 @@ build:
 
 requirements:
   build:
-    - python >=3.5
+    - python >=3.6
```

Comment:
Should we drop Python 3.6? Asking as RAPIDS and Dask have now dropped it

Reply:
Yeah that probably makes sense